testing/trino-tests/src/test/java/io/trino/execution/TestCoordinatorDynamicFiltering.java

```diff
@@ -215,17 +209,6 @@ public void testIsNotDistinctFromJoinWithSelectiveBuildSide()
                         Domain.onlyNull(BIGINT))));
     }
 
-    @Test(timeOut = 30_000)
-    public void testBroadcastJoinWithSelectiveBuildSide()
```

Comment:
why it's removed?

Reply:
It's redundant with the parameterized `testJoinWithEmptyBuildSide` now

Reply:
it's actually redundant with parameterised testJoinWithSelectiveBuildSide now